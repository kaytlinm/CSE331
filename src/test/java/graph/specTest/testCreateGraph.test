#Create Graph Test File
#Tests that a Graph can be created

CreateGraph g1
