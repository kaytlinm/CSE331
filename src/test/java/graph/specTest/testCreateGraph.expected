#Create Graph Test File
#Tests that a Graph can be created

created graph g1